### ISSUE TITLE
Content: Accessibility Lesson 4 has a wrong title

### ISSUE THREAD

**Issue by marcogonzalo** [learnpack/learnpack#1880]
**Describe the bug**
It's called as " Insufficient Contrast" but next pages are talking about other errors that are not about contrast. Maybe, a title like "Commo errors in accessibility" would be better. And chengin the next ones...
**In what exercise**
04.0 Insufficient Contrast
